tracking/src/main/java/org/motechproject/nms/tracking/service/CacheService.java

```diff
@@ -0,0 +1,5 @@
+package org.motechproject.nms.tracking.service;
+
+public interface CacheService {
```

Comment:
Is there a reason you created the cache service bean?  My concern is if we want to cache the results of another method we should just be able to add the @Cacheable annotation and not need to add a new method to this bean